components/Makefile

```diff
@@ -23,6 +23,9 @@
 
 -include ../make-rules/shared-macros.mk
 
+# If we need to build incorporations on the build system where the final repo is not the same as the our local repo, we need to specify the INCORPORATION_REPO variable.
```

Comment:
Please make this line shorter (multiline comments are common here).   There is no reason to have a comment with almost 170 characters in one line.  Thank you.